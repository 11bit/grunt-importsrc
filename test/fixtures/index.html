<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>grunt-importsrc</title>

  <!-- importsrc concat:temp/styles/importsrcs-concat_output.css -->
  <link rel="stylesheet" href="styles/reset.css">
  <link rel="stylesheet" href="styles/main.css">
  <!-- endimportsrc -->

  <!-- importsrc update:mincss.test.src -->
  <link rel="stylesheet" href="styles/reset.css">
  <link rel="stylesheet" href="styles/main.css">
  <!-- endimportsrc -->

  <!-- importsrc
    update:mincss.test_without_dest.src
    dest:temp/styles/mincss_without_dest_output.css -->
  <link rel="stylesheet" href="styles/input.css">
  <link rel="stylesheet" href="styles/reset.css">
  <link rel="stylesheet" href="styles/main.css">
  <!-- endimportsrc -->
</head>
<body>

  <h1>Hello world</h1>

  <!-- importsrc
    concat:temp/scripts/importsrcs-concat_output.js
    replace:test_fake_replace_path.js -->
  <script src="scripts/vendors/plugin.js"></script>
  <script src="scripts/file-1.js"></script>
  <script src="scripts/file-2.js"></script>
  <!-- endimportsrc -->

  <!--
    importsrc
    update:uglify.test.files['temp/scripts/uglify_output.js']
  -->
  <script src="scripts/vendors/plugin.js"></script>
  <script src="scripts/file-1.js"></script>
  <script src="scripts/file-2.js"></script>
  <!-- endimportsrc -->

</body>
</html>
