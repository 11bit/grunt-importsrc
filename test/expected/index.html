<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>grunt-importsrc</title>

  <link rel="stylesheet" href="temp/styles/importsrcs-concat_output.css">

  <link rel="stylesheet" href="temp/styles/mincss_output.css">

  <link rel="stylesheet" href="temp/styles/mincss_without_dest_output.css">
</head>
<body>

  <h1>Hello world</h1>

  <script src="temp/scripts/importsrcs-concat_output.js"></script>

  <script src="temp/scripts/uglify_output.js"></script>

</body>
</html>
